django/contrib/gis/gdal/raster/band.py

```diff
@@ -47,19 +56,80 @@ def pixel_count(self):
         """
         return self.width * self.height
 
+    _stats_refresh = False
+
+    def statistics(self, refresh=False, approximate=False):
+        """
+        Compute statistics on the pixel values of this band.
+
+        The return value is a tuple with the following structure:
+        (minimum, maximum, mean, standard deviation).
+
+        If approximate=True, the statistics may be computed based overviews, or
```

Comment:
"computed based overviews" is there a missing word? e.g. "computed based on overviews"?
I'm not sure of the exact meaning, but probably chop the comma before "or".